File: app/src/main/java/com/example/mvctutorial/designpattern/_08absttractfactory_guifactory/windows/WindowsTextArea.java
package com.example.mvctutorial.designpattern._08absttractfactory_guifactory.windows;

import com.example.mvctutorial.designpattern._08absttractfactory_guifactory.TextArea;

public class WindowsTextArea implements TextArea {
    @Override
    public String getText() {
        return "windows text area";
    }
}

File: app/src/main/java/com/example/mvctutorial/designpattern/_08absttractfactory_guifactory/windows/WindowsTextArea.kt
package com.example.mvctutorial.designpattern._08absttractfactory_guifactory.windows

import com.example.mvctutorial.designpattern._08absttractfactory_guifactory.TextArea

class WindowsTextArea: TextArea {
    override fun getText(): String = "windows text area"
}